projects/packages/sync/src/class-actions.php

```diff
@@ -194,6 +194,13 @@ public static function should_initialize_sender() {
 			return self::sync_via_cron_allowed();
 		}
 
+		/**
+		 * For now, if Sync spawning is enabled we will always initialize send, even for GET and unauthenticated requests.
+		 */
+		if ( Settings::is_sync_spawning_enabled() ) {
```

Comment:
I'm wondering if `spawning` is descriptive enough in this case.

Wouldn't it be a bit better to be a bit more descriptive like `sync_dedicated_request` or `sync_dedicated_send_request` or something 🤔